extensions/iterable/v1/actions/sendEmail/config/fields.ts

```diff
@@ -0,0 +1,122 @@
+import { z, type ZodTypeAny } from 'zod'
+import {
+  type Field,
+  FieldType,
+  NumericIdSchema,
+  makeStringOptional,
+} from '@awell-health/extensions-core'
+import { isEmpty, isNil } from 'lodash'
+import { differenceInDays, formatISO9075, isAfter, parseJSON } from 'date-fns'
+import { StringTransformToJson } from '../../../validation'
+
+export const fields = {
+  campaignId: {
+    label: 'Campaign ID',
+    id: 'campaignId',
+    type: FieldType.NUMERIC,
+    required: true,
+    description: 'Campaign ID in Iterable.',
```

Comment:
Not a super valuable description.

```suggestion
    description: '',
```